### PR TITLE
Add alternate language links to head, enables Global Nav lang switcher

### LINKED ISSUE #94: Localized version of page should load the correct header
It's currently loading the header in english.

Page: https://main--esri--aemsites.hlx.page/ja-jp/about/about-esri/overview
Live site: https://www.esri.com/ja-jp/about/about-esri/overview

### PR DESCRIPTION
**Pull request by twhite313**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #94

Test URLs:
- Original: https://www.esri.com/en-us/about/about-esri/overview
- Before: https://main--esri--aemsites.hlx.live/
- After: https://lang--esri--aemsites.hlx.live/

### CODE REVIEW

**Comment by twhite313** [comment]
@alexcarol not sure if its the best place to add it, but I have alternate langauges links being added to the head, which enables the language switcher. (The icon is misaligned for some reason, but that is for another task.)

I can't get past some of the linter errors. One makes no sense: it says splitPattern isn't used, but it clearly is 4 lines later.  I tried modifying some of the other issues but it just ends up breaking my code.

File: scripts/scripts.js [hunk 1]
```diff
@@ -44,6 +45,62 @@ async function loadFonts() {
   }
 }
 
+/**
```
Comment (alexcarol) [inline scripts/scripts.js:48]: As this is part of the header logic, could you move it to "header.js"? As a note, anything in scripts.js (or most files for that matter) should be encapsulated in a function (not just called when scripts.js is included).

**Comment by alexcarol** [comment]
> @alexcarol not sure if its the best place to add it, but I have alternate langauges links being added to the head, which enables the language switcher. (The icon is misaligned for some reason, but that is for another task.)
> 
> I can't get past some of the linter errors. One makes no sense: it says splitPattern isn't used, but it clearly is 4 lines later. I tried modifying some of the other issues but it just ends up breaking my code.

I don't see "splitPattern" used anywhere, are you looking for it in the diff or in your editor? (in case there's changes that might not be pushed). Can you share a link to the line where it's used?

**Comment by alexcarol** [comment]
@twhite313 here's a suggested change for your PR: https://github.com/aemsites/esri/pull/135
Feel free to use it or change as you need

**Comment by twhite313** [comment]
> > @alexcarol not sure if its the best place to add it, but I have alternate langauges links being added to the head, which enables the language switcher. (The icon is misaligned for some reason, but that is for another task.)
> > I can't get past some of the linter errors. One makes no sense: it says splitPattern isn't used, but it clearly is 4 lines later. I tried modifying some of the other issues but it just ends up breaking my code.
> 
> I don't see "splitPattern" used anywhere, are you looking for it in the diff or in your editor? (in case there's changes that might not be pushed). Can you share a link to the line where it's used?

Well that's really weird. You and the linter couldn't see it. Literally there on line 60 and 64. Whatever; moving on. 

split

**Comment by alexcarol** [comment]
> > > @alexcarol not sure if its the best place to add it, but I have alternate langauges links being added to the head, which enables the language switcher. (The icon is misaligned for some reason, but that is for another task.)
> > > I can't get past some of the linter errors. One makes no sense: it says splitPattern isn't used, but it clearly is 4 lines later. I tried modifying some of the other issues but it just ends up breaking my code.
> > 
> > 
> > I don't see "splitPattern" used anywhere, are you looking for it in the diff or in your editor? (in case there's changes that might not be pushed). Can you share a link to the line where it's used?
> 
> Well that's really weird. You and the linter couldn't see it. Literally there on line 60 and 64. Whatever; moving on.
> 
> split
> 
> Oh! The error changed. I see it is failing AFTER line 64 now. I was getting errors that it was never declared. Anyway... moving on.

The issue was that it was assigned, but never read, so it's essentially not "used".

**Pull request merged**